### DISCUSSION TITLE
Dudas frente al proyecto final (Estudiante NO UDEA)

### DISCUSSION THREAD

**Discussion by MarcosArangoTamayo** [rramosp/ai4eng.v1#218] [00 - General]
Hola, me recomendaron el curso y lo comencé, me ha gustado bastante, los videos son de gran ayuda y están resumidos.

No soy estudiante de la UDEA, los lineamientos dicen que se debe alcanzar el 90% de la nota. Según veo en la plataforma, la calificación le atribuye un 2.0 al proyecto de analítica y 0.5 a las entregas previas del proyecto. Mi duda es si esas dos notas corresponden exclusivamente a los estudiantes que tengan registrado la materia en la UDEA, o es para todos los que estén llevando a cabo el curso abierto.  

De ser el segundo caso, ¿Dónde aparecen los lineamientos para ese proyecto?

**Comment by rramosp** [comment]
hola marcos, debes de apuntarte a la versión abierta, no a la de la udea

On Sat, Feb 11, 2023 at 12:13 AM MarcosArangoTamayo <
***@***.***> wrote:

> Hola, me recomendaron el curso y lo comencé, me ha gustado bastante, los
> videos son de gran ayuda y están resumidos.
>
> No soy estudiante de la UDEA, los lineamientos dicen que se debe alcanzar
> el 90% de la nota. Según veo en la plataforma, la calificación le atribuye
> un 2.0 al proyecto de analítica y 0.5 a las entregas previas del proyecto.
> Mi duda es si esas dos notas corresponden exclusivamente a los estudiantes
> que tengan registrado la materia en la UDEA, o es para todos los que estén
> llevando a cabo el curso abierto.
>
> De ser el segundo caso, ¿Dónde aparecen los lineamientos para ese proyecto?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/218>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZILTSIZDMPVH6KQXIGLWW4NXBANCNFSM6AAAAAAUYP4UQM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
